src/netius/extra/proxy_c.py

```
@@ -0,0 +1,278 @@
+            # cleaned up on the next rebuild cycle
+            self._consul_hosts.add(domain)
+
+            self.debug(
+                "Registered Consul service '%s' as '%s' with %d instance(s)"
+                % (service, domain, len(urls))
+            )
+
+    def _build_suffixes(self, alias=True, redirect=True):
+        for host_suffix in self.host_suffixes:
+            self.info("Registering %s host suffix" % host_suffix)
+            for _alias, value in netius.legacy.items(self.alias):
```

Pylint (4.0.4)

[error] Instance of 'ConsulProxyServer' has no 'alias' member

(E1101)